projects/1.19/assets/caffeinefilled/caffeinefilled/lang/zh_cn.json

```
@@ -0,0 +1,74 @@
+{
+  "advancements.average_drink.descr": "喝一杯常见的咖啡",
+  "advancements.average_drink.title": "一般饮料",
+  "advancements.babys_taste.descr": "喝一杯淡咖啡",
+  "advancements.babys_taste.title": "婴儿的品味",
+  "advancements.full_coffeine.descr": "准备任意一杯咖啡",
+  "advancements.full_coffeine.title": "满溢的咖啡因",
+  "advancements.kookie_coffee_advancement.descr": "喝下Kookie咖啡",
+  "advancements.kookie_coffee_advancement.title": "向Kookie.com.br致敬",
+  "advancements.never_sleep.descr": "喝下一杯特浓咖啡",
+  "advancements.never_sleep.title": "从未睡觉",
+  "advancements.pinkarella_special_advancement.descr": "喝下特别粉色天使",
+  "advancements.pinkarella_special_advancement.title": "向《屠宰马》致敬",
+  "block.caffeinefilled.andesite_counter": "安山岩台面",
+  "block.caffeinefilled.blackstone_counter": "黑石柜台",
+  "block.caffeinefilled.brick_counter": "砖砌台面",
+  "block.caffeinefilled.coffee_bean_bag": "袋装咖啡豆",
+  "block.caffeinefilled.coffee_berries_bag": "袋装咖啡果",
+  "block.caffeinefilled.coffee_bush_phase_0": "咖啡丛生长阶段 0",
+  "block.caffeinefilled.coffee_bush_phase_1": "咖啡丛生长阶段 0",
+  "block.caffeinefilled.coffee_bush_phase_2": "咖啡丛生长阶段 0",
+  "block.caffeinefilled.coffee_bush_phase_3": "咖啡丛生长阶段 0",
+  "block.caffeinefilled.deepslate_counter": "黑石柜台/深板岩柜台",
+  "block.caffeinefilled.diorite_counter": "闪长岩台面",
+  "block.caffeinefilled.granite_counter": "花岗岩台面",
+  "block.caffeinefilled.lemon_crate": "箱装柠檬",
+  "item.caffeinefilled.afogatto": "阿芙佳朵",
+  "item.caffeinefilled.americano": "美式咖啡",
+  "item.caffeinefilled.black_coffee": "黑咖啡",
+  "item.caffeinefilled.black_eye": "黑眼咖啡",
+  "item.caffeinefilled.ca_phe_sua_da": "越式冰咖啡",
+  "item.caffeinefilled.cafe_au_lait": "欧蕾咖啡",
+  "item.caffeinefilled.cafe_com_cheirinho": "威士忌黑咖啡",
+  "item.caffeinefilled.caffe_tobio": "Tobio咖啡",
+  "item.caffeinefilled.cappuccino": "卡布奇诺",
+  "item.caffeinefilled.carioca": "西爱西咖啡/里约热内卢咖啡",
+  "item.caffeinefilled.carioca_de_limao": "柠檬西爱西咖啡/柠檬里约热内卢咖啡",
+  "item.caffeinefilled.chocolate": "巧克力",
+  "item.caffeinefilled.chocolate_ice_cream_bowl": "巧克力冰淇淋",
+  "item.caffeinefilled.coffee_bean": "烤咖啡豆",
+  "item.caffeinefilled.coffee_berries": "咖啡果",
+  "item.caffeinefilled.coffee_ice_cream_bowl": "咖啡冰淇淋",
+  "item.caffeinefilled.cold_brew": " 冷萃咖啡",
+  "item.caffeinefilled.cortado": "可塔朵",
+  "item.caffeinefilled.cream": "奶油",
+  "item.caffeinefilled.doppio": "双份浓缩咖啡",
+  "item.caffeinefilled.egg_coffee": "蛋蜜咖啡",
```

```suggestion
  "item.caffeinefilled.egg_coffee": "蛋咖啡",
```
https://en.wikipedia.org/wiki/Egg_coffee